.github/workflows/ci.yaml

```diff
@@ -12,9 +12,9 @@ jobs:
     strategy:
       matrix:
         include:
-          - python_version: "3.7"
-            variant: stretch
-            test_project: django2
+          # - python_version: "3.7"
+          #   variant: stretch
+          #   test_project: django2
```

Comment:
Should we maybe just remove this entirely? Since 3.7 is no longer supported, we don't need to build newer versions of this image.

Reply:
We should. That whole build (which uses the wheelhouse) also needs to go away, but that's for a different PR.